include/rocksdb/utilities/table_properties_collectors.h

```diff
@@ -38,6 +38,8 @@ class CompactOnDeletionCollectorFactory
     return "CompactOnDeletionCollector";
   }
 
+  virtual std::string ToString() const override;
```

Comment:
Minor: could you please change this and all overriding methods (including the destructor) so that `virtual` is not explicitly specified but `override` is?

Reply:
I think this class is extendable - specifically, MyRocks is planning to extend and override some methods so they can track such compactions.

Reply:
I'm not saying these methods shouldn't be virtual; it's just a coding style comment (see https://google.github.io/styleguide/cppguide.html#Inheritance). `override` in the derived class implies `virtual` (the code does not compile if it doesn't override anything), so `virtual` in the derived class is superfluous. (In the base class, `virtual` is of course necessary.)